support/kind/kind.go

```diff
@@ -69,17 +70,17 @@ func (k *Cluster) getKubeconfig() (string, error) {
 }
 
 func (k *Cluster) CreateWithConfig(imageName, kindConfigFile string) (string, error) {
-	log.Println("Creating kind cluster ", k.name)
+	log.Info("Creating kind cluster ", k.name)
 	if err := k.findOrInstallKind(k.e); err != nil {
 		return "", err
 	}
 
 	if strings.Contains(k.e.Run("kind get clusters"), k.name) {
-		log.Println("Skipping Kind Cluster.Create: cluster already created: ", k.name)
+		log.Info("Skipping Kind Cluster.Create: cluster already created: ", k.name)
 		return k.getKubeconfig()
 	}
 
-	log.Println("launching: kind create cluster --name", k.name, "--image", imageName, "--config", kindConfigFile)
+	log.Info("launching: kind create cluster --name", k.name, "--image", imageName, "--config", kindConfigFile)
```

Comment:
Change to at least level 4 here, and the rest of the locations in this file, for trace/debug.